tests/buyers/views/test_buyers.py

```diff
@@ -1964,7 +1973,10 @@ def test_page_does_not_pluralise_for_single_response(self, data_api_client):
                 api_stubs.lot(slug='digital-outcomes', allows_brief=True),
             ]
         )
-        data_api_client.get_brief.return_value = api_stubs.brief(lot_slug="digital-outcomes")
+
+        brief_stub = api_stubs.brief(lot_slug="digital-outcomes")
+        brief_stub['briefs'].update({'publishedAt': '2017-01-21T12:00:00.000000Z'})
+        data_api_client.get_brief.return_value = brief_stub
```

Comment:
It feels like all these tests have a lot of duplication in terms of their set up (not just the stuff you've added but generally). Lots of frameworks and briefs being set repeatedly. You could move some of these into a setup method